website/client/assets/images/world-boss/check.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="13" height="10" viewBox="0 0 13 10">
```

Comment:
this should go in /svg not /images or the webpack loader won't work correctly

Reply:
whoops! OK, will fix that on the merged branch